Cargo.toml

```
@@ -9,6 +9,10 @@ build = "build.rs"
 edition = "2018"
 
 [workspace]
+exclude = [
+    "ignored",
+    ".workspace",
+]
```

Can we instead use `members =` so we don't have to worry about any other folders the user has?
```suggestion
members = ["metadata"]
```

---

Dunno why, but members doesn't work.

---

This is https://github.com/rust-lang/cargo/issues/5418, it seems like there's no way to override this 🤦

---

Ok, in that case looks good once the docker workspace is ignored too.
```suggestion
exclude = [
    "ignored",
    ".workspace",
    ".rustwide-docker",
]
```